### ISSUE TITLE
Error when restoring default config to device

### ISSUE THREAD

**Issue by bocklucas** [appdevelopmentandsuch/AnotherTempSensor#11]
**Summary**

The following error gets thrown on the device when the `Flash` button is pushed (it is used to restore the default configuration to the device)

```
--------------- CUT HERE FOR EXCEPTION DECODER ---------------

Exception (9):
epc1=0x401052b4 epc2=0x00000000 epc3=0x00000000 excvaddr=0x00000003 depc=0x00000000

>>>stack>>>

ctx: sys
sp: 3fffeb40 end: 3fffffb0 offset: 0190
3fffecd0:  3ffe8600 0000001f 3fff0bf4 3fff0c7d  
3fffece0:  401054e5 00b06f28 3ffeecc8 00000000  
3fffecf0:  3ffee230 3ffeecc8 3ffe868b 3fff00cc  
3fffed00:  00000000 00aff516 4020d118 00000064  
3fffed10:  3ffeecc8 ffffffff 00000020 4020d126  
3fffed20:  ffffffff 3ffeeaf4 3ffeeaf4 40100175  
3fffed30:  40100486 00000000 00000000 40100510  
3fffed40:  e0037035 00000000 00000000 3ffea4a8  
3fffed50:  00000d90 00000000 00000020 00000022  
3fffed60:  3fffc200 4010044c 3fffc258 4000050c  
3fffed70:  4023d2f5 00000030 00000020 ffffffff  
3fffed80:  4023d2f5 60000a04 00000001 00000000  
3fffed90:  00000028 03280062 01280062 00000004  
3fffeda0:  01280062 00064009 fffffffd 3ffee6a0  
3fffedb0:  60000600 60000200 3ffee7fa 00000030  
3fffedc0:  40229b69 3fffee80 3ffedb48 00000016  
3fffedd0:  3ffed4bc 3fffee80 3ffedb48 00000016  
3fffede0:  402275bc 3fffee80 3ffed9f4 3ffed3ec  
3fffedf0:  4022761d 3fffee80 3fffee80 ffffff80  
3fffee00:  4f704875 02004762 40102a86 00000100  
3fffee10:  3ffe97a0 7fffffff 00000000 00000001  
3fffee20:  40223925 00000080 3ffee6b0 00000000  
3fffee30:  00000000 00000000 0000001f 40100230  
3fffee40:  3ffee230 3ffef48c 3fffc228 401053a1  
3fffee50:  4023d27c 3ffee6a0 60000600 60000200  
3fffee60:  4023d2cf 3ffee6a0 60000600 ffffffff  
3fffee70:  4023c0e7 00000000 3ffee6b0 00000000  
3fffee80:  40106286 60000e00 3ffe8d34 bfffffff  
3fffee90:  40240f38 3ffef1e4 3ffe8d34 3fffdab0  
3fffeea0:  0000000a 3fffdcb0 3ffee240 00000000  
3fffeeb0:  00000000 00000000 022907e2 40244320  
3fffeec0:  00000008 00000000 f7ffffff 60000600  
3fffeed0:  00000001 00000000 00000000 40217108  
3fffeee0:  4023416c 00000008 3ffee230 022906a4  
3fffeef0:  40234572 0228fe1a 60000600 40217173  
3fffef00:  402309e0 3ffe8d34 40234637 3ffed28c  
3fffef10:  3ffee230 00acb020 3ffee230 60000600  
3fffef20:  40237f81 3ffed28c 3ffee230 0228fe1a  
3fffef30:  40237fc6 3fffdab0 00000000 3fffdcb0  
3fffef40:  3ffee248 40000f65 3fffdab0 00000000  
3fffef50:  40000f49 3ffe8fe0 3fffdab0 40000f49  
3fffef60:  40000e19 00000005 0004efec 00000000  
3fffef70:  00000000 aa55aa55 00000094 40104c01  
3fffef80:  40104c07 0004efec 00000000 02320102  
3fffef90:  4010000d 70103380 c7721470 935670fd  
3fffefa0:  4022fc08 3fffef3c 4022fbc1 3fffff28  
3fffefb0:  3fffffc0 00000000 00000000 feefeffe  
3fffefc0:  feefeffe feefeffe feefeffe feefeffe  
3fffefd0:  feefeffe feefeffe feefeffe feefeffe  
3fffefe0:  feefeffe feefeffe feefeffe feefeffe  
3fffeff0:  feefeffe feefeffe feefeffe feefeffe  
3ffff000:  feefeffe feefeffe feefeffe feefeffe  
3ffff010:  feefeffe feefeffe feefeffe feefeffe  
3ffff020:  feefeffe feefeffe feefeffe feefeffe  
3ffff030:  feefeffe feefeffe feefeffe feefeffe  
3ffff040:  feefeffe feefeffe feefeffe feefeffe  
3ffff050:  feefeffe feefeffe feefeffe feefeffe  
3ffff060:  feefeffe feefeffe feefeffe feefeffe  
3ffff070:  feefeffe feefeffe feefeffe feefeffe  
3ffff080:  feefeffe feefeffe feefeffe feefeffe  
3ffff090:  feefeffe feefeffe feefeffe feefeffe  
3ffff0a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff0b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff0c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff0d0:  feefeffe feefeffe feefeffe feefeffe  
3ffff0e0:  feefeffe feefeffe feefeffe feefeffe  
3ffff0f0:  feefeffe feefeffe feefeffe feefeffe  
3ffff100:  feefeffe feefeffe feefeffe feefeffe  
3ffff110:  feefeffe feefeffe feefeffe feefeffe  
3ffff120:  feefeffe feefeffe feefeffe feefeffe  
3ffff130:  feefeffe feefeffe feefeffe feefeffe  
3ffff140:  feefeffe feefeffe feefeffe feefeffe  
3ffff150:  feefeffe feefeffe feefeffe feefeffe  
3ffff160:  feefeffe feefeffe feefeffe feefeffe  
3ffff170:  feefeffe feefeffe feefeffe feefeffe  
3ffff180:  feefeffe feefeffe feefeffe feefeffe  
3ffff190:  feefeffe feefeffe feefeffe feefeffe  
3ffff1a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff1b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff1c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff1d0:  feefeffe feefeffe feefeffe feefeffe  
3ffff1e0:  feefeffe feefeffe feefeffe feefeffe  
3ffff1f0:  feefeffe feefeffe feefeffe feefeffe  
3ffff200:  feefeffe feefeffe feefeffe feefeffe  
3ffff210:  feefeffe feefeffe feefeffe feefeffe  
3ffff220:  feefeffe feefeffe feefeffe feefeffe  
3ffff230:  feefeffe feefeffe feefeffe feefeffe  
3ffff240:  feefeffe feefeffe feefeffe feefeffe  
3ffff250:  feefeffe feefeffe feefeffe feefeffe  
3ffff260:  feefeffe feefeffe feefeffe feefeffe  
3ffff270:  feefeffe feefeffe feefeffe feefeffe  
3ffff280:  feefeffe feefeffe feefeffe feefeffe  
3ffff290:  feefeffe feefeffe feefeffe feefeffe  
3ffff2a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff2b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff2c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff2d0:  feefeffe feefeffe feefeffe feefeffe  
3ffff2e0:  feefeffe feefeffe feefeffe feefeffe  
3ffff2f0:  feefeffe feefeffe feefeffe feefeffe  
3ffff300:  feefeffe feefeffe feefeffe feefeffe  
3ffff310:  feefeffe feefeffe feefeffe feefeffe  
3ffff320:  feefeffe feefeffe feefeffe feefeffe  
3ffff330:  feefeffe feefeffe feefeffe feefeffe  
3ffff340:  feefeffe feefeffe feefeffe feefeffe  
3ffff350:  feefeffe feefeffe feefeffe feefeffe  
3ffff360:  feefeffe feefeffe feefeffe feefeffe  
3ffff370:  feefeffe feefeffe feefeffe feefeffe  
3ffff380:  feefeffe feefeffe feefeffe feefeffe  
3ffff390:  feefeffe feefeffe feefeffe feefeffe  
3ffff3a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff3b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff3c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff3d0:  feefeffe feefeffe feefeffe feefeffe  
3ffff3e0:  feefeffe feefeffe feefeffe feefeffe  
3ffff3f0:  feefeffe feefeffe feefeffe feefeffe  
3ffff400:  feefeffe feefeffe feefeffe feefeffe  
3ffff410:  feefeffe feefeffe feefeffe feefeffe  
3ffff420:  feefeffe feefeffe feefeffe feefeffe  
3ffff430:  feefeffe feefeffe feefeffe feefeffe  
3ffff440:  feefeffe feefeffe feefeffe feefeffe  
3ffff450:  feefeffe feefeffe feefeffe feefeffe  
3ffff460:  feefeffe feefeffe feefeffe feefeffe  
3ffff470:  feefeffe feefeffe feefeffe feefeffe  
3ffff480:  feefeffe feefeffe feefeffe feefeffe  
3ffff490:  feefeffe feefeffe feefeffe feefeffe  
3ffff4a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff4b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff4c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff4d0:  feefeffe feefeffe feefeffe feefeffe  
3ffff4e0:  feefeffe feefeffe feefeffe feefeffe  
3ffff4f0:  feefeffe feefeffe feefeffe feefeffe  
3ffff500:  feefeffe feefeffe feefeffe feefeffe  
3ffff510:  feefeffe feefeffe feefeffe feefeffe  
3ffff520:  feefeffe feefeffe feefeffe feefeffe  
3ffff530:  feefeffe feefeffe feefeffe feefeffe  
3ffff540:  feefeffe feefeffe feefeffe feefeffe  
3ffff550:  feefeffe feefeffe feefeffe feefeffe  
3ffff560:  feefeffe feefeffe feefeffe feefeffe  
3ffff570:  feefeffe feefeffe feefeffe feefeffe  
3ffff580:  feefeffe feefeffe feefeffe feefeffe  
3ffff590:  feefeffe feefeffe feefeffe feefeffe  
3ffff5a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff5b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff5c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff5d0:  feefeffe feefeffe feefeffe feefeffe  
3ffff5e0:  feefeffe feefeffe feefeffe feefeffe  
3ffff5f0:  feefeffe feefeffe feefeffe feefeffe  
3ffff600:  feefeffe feefeffe feefeffe feefeffe  
3ffff610:  feefeffe feefeffe feefeffe feefeffe  
3ffff620:  feefeffe feefeffe feefeffe feefeffe  
3ffff630:  feefeffe feefeffe feefeffe feefeffe  
3ffff640:  feefeffe feefeffe feefeffe feefeffe  
3ffff650:  feefeffe feefeffe feefeffe feefeffe  
3ffff660:  feefeffe feefeffe feefeffe feefeffe  
3ffff670:  feefeffe feefeffe feefeffe feefeffe  
3ffff680:  feefeffe feefeffe feefeffe feefeffe  
3ffff690:  feefeffe feefeffe feefeffe feefeffe  
3ffff6a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff6b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff6c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff6d0:  feefeffe feefeffe feefeffe feefeffe  
3ffff6e0:  feefeffe feefeffe feefeffe feefeffe  
3ffff6f0:  feefeffe feefeffe feefeffe feefeffe  
3ffff700:  feefeffe feefeffe feefeffe feefeffe  
3ffff710:  feefeffe feefeffe feefeffe feefeffe  
3ffff720:  feefeffe feefeffe feefeffe feefeffe  
3ffff730:  feefeffe feefeffe feefeffe feefeffe  
3ffff740:  feefeffe feefeffe feefeffe feefeffe  
3ffff750:  feefeffe feefeffe feefeffe feefeffe  
3ffff760:  feefeffe feefeffe feefeffe feefeffe  
3ffff770:  feefeffe feefeffe feefeffe feefeffe  
3ffff780:  feefeffe feefeffe feefeffe feefeffe  
3ffff790:  feefeffe feefeffe feefeffe feefeffe  
3ffff7a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff7b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff7c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff7d0:  feefeffe feefeffe feefeffe feefeffe  
3ffff7e0:  feefeffe feefeffe feefeffe feefeffe  
3ffff7f0:  feefeffe feefeffe feefeffe feefeffe  
3ffff800:  feefeffe feefeffe feefeffe feefeffe  
3ffff810:  feefeffe feefeffe feefeffe feefeffe  
3ffff820:  feefeffe feefeffe feefeffe feefeffe  
3ffff830:  feefeffe feefeffe feefeffe feefeffe  
3ffff840:  feefeffe feefeffe feefeffe feefeffe  
3ffff850:  feefeffe feefeffe feefeffe feefeffe  
3ffff860:  feefeffe feefeffe feefeffe feefeffe  
3ffff870:  feefeffe feefeffe feefeffe feefeffe  
3ffff880:  feefeffe feefeffe feefeffe feefeffe  
3ffff890:  feefeffe feefeffe feefeffe feefeffe  
3ffff8a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff8b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff8c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff8d0:  feefeffe feefeffe feefeffe feefeffe  
3ffff8e0:  feefeffe feefeffe feefeffe feefeffe  
3ffff8f0:  feefeffe feefeffe feefeffe feefeffe  
3ffff900:  feefeffe feefeffe feefeffe feefeffe  
3ffff910:  feefeffe feefeffe feefeffe feefeffe  
3ffff920:  feefeffe feefeffe feefeffe feefeffe  
3ffff930:  feefeffe feefeffe feefeffe feefeffe  
3ffff940:  feefeffe feefeffe feefeffe feefeffe  
3ffff950:  feefeffe feefeffe feefeffe feefeffe  
3ffff960:  feefeffe feefeffe feefeffe feefeffe  
3ffff970:  feefeffe feefeffe feefeffe feefeffe  
3ffff980:  feefeffe feefeffe feefeffe feefeffe  
3ffff990:  feefeffe feefeffe feefeffe feefeffe  
3ffff9a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff9b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff9c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff9d0:  feefeffe feefeffe feefeffe feefeffe  
3ffff9e0:  feefeffe feefeffe feefeffe feefeffe  
3ffff9f0:  feefeffe feefeffe feefeffe feefeffe  
3ffffa00:  00000002 00000000 00000020 40100230  
3ffffa10:  40101c81 feefeffe 00000002 40101850  
3ffffa20:  3ffe8f22 40104ae3 3ffec690 feefeffe  
3ffffa30:  00000001 40103cbe 3ffed0e8 feefeffe  
3ffffa40:  4010412f feefeffe feefeffe feefeffe  
3ffffa50:  feefeffe feefeffe feefeffe feefeffe  
3ffffa60:  40103bf7 0000001e 00000002 00040000  
3ffffa70:  00002200 feefeffe feefeffe feefeffe  
3ffffa80:  40102ac0 00080000 feefeffe feefeffe  
3ffffa90:  feefeffe 00000000 feefeffe feefeffe  
3ffffaa0:  feefeffe 2c9f0300 4000050c 3fffc278  
3ffffab0:  401027a8 3fffc200 00000022 feefeffe  
3ffffac0:  4022a7e1 00000030 00000008 ffffffff  
3ffffad0:  40220f27 3ffef8dc 3ffec6b8 00000000  
3ffffae0:  00000006 0000005f 40245174 00000002  
3ffffaf0:  3ffebd90 00000000 00000020 3ffef8dc  
3ffffb00:  3ffedb48 00000000 3ffef474 00000030  
3ffffb10:  feefeffe feefeffe feefeffe feefeffe  
3ffffb20:  feefeffe feefeffe feefeffe feefeffe  
3ffffb30:  feefeffe feefeffe feefeffe feefeffe  
3ffffb40:  feefeffe feefeffe feefeffe feefeffe  
3ffffb50:  feefeffe feefeffe feefeffe feefeffe  
3ffffb60:  feefeffe feefeffe feefeffe feefeffe  
3ffffb70:  feefeffe feefeffe 40231e7b 00000001  
3ffffb80:  ffffffff 00000000 3ffe9071 00000008  
3ffffb90:  40231eca 3ffec690 3ffef474 00000001  
3ffffba0:  00000002 00000000 40231e7b 00000001  
3ffffbb0:  ffffffff 00000000 3ffe9071 00000008  
3ffffbc0:  40231eca 3ffec6b8 3ffef474 00000001  
3ffffbd0:  40231fd6 3ffec6b8 3ffef474 3ffec6b8  
3ffffbe0:  000005e0 00000000 40103901 3ffed0a0  
3ffffbf0:  0000003c 00000000 3ffec6b8 00000100  
3ffffc00:  00000025 00000001 40104a49 3ffed0a0  
3ffffc10:  3ffed070 00000002 40231e7b 00000001  
3ffffc20:  00000001 00000000 00000014 40100230  
3ffffc30:  40231eca 3ffec6b8 00000001 402359f3  
3ffffc40:  40231fae 3ffec6b8 4010198b 3ffec6b8  
3ffffc50:  00000014 00000020 00000008 3fff0110  
3ffffc60:  40230f3c 3ffec6b8 3ffece30 00000020  
3ffffc70:  00000000 40224543 3ffedb48 3ffef474  
3ffffc80:  00000000 00000002 00000000 3ffec6b8  
3ffffc90:  00000003 00000000 00000010 00000000  
3ffffca0:  00000003 00000000 00000010 00000000  
3ffffcb0:  3ffe97c4 4021589d 3fff0bb4 3ffeee34  
3ffffcc0:  00000000 80000000 00000000 0000001b  
3ffffcd0:  00001800 00000000 00001800 00000000  
3ffffce0:  3ffffe40 00000000 3ffffe40 4021523a  
3ffffcf0:  00008000 3ffffde3 3fff00f4 40215f48  
3ffffd00:  3ffef0b0 00000030 402112b9 00000002  
3ffffd10:  3ffffe40 00000002 3ffffe40 4021523a  
3ffffd20:  3ffffda0 3ffffde1 3ffffd50 00000000  
3ffffd30:  0000000a 3fffc6fc 00000001 40215170  
3ffffd40:  3ffffe40 3ffffde1 3ffffda0 402113e4  
3ffffd50:  00000002 00000213 00000000 00000001  
3ffffd60:  3ffe8300 00000000 00000010 4024628f  
3ffffd70:  3ffffde3 00000003 00000000 00000000  
3ffffd80:  3fff0138 3fff06e8 3ffef220 3ffe8af0  
3ffffd90:  00000000 3ffe8aef 3ffffe40 4021566b  
3ffffda0:  00000004 ffffffff ffffffff 00000002  
3ffffdb0:  00000002 00000011 00302058 4021fc08  
3ffffdc0:  00000000 3ffeecc8 00000014 3fff06e4  
3ffffdd0:  3fff07dc 3ffef220 00000000 4021f158  
3ffffde0:  00000d90 00005800 00000000 4021a545  
3ffffdf0:  00000000 00001b53 00000000 00000000  
3ffffe00:  000012c8 00000020 00000020 40100ca4  
3ffffe10:  00000002 00000009 00000001 00000000  
3ffffe20:  00001780 000002f0 000002f0 401009e7  
3ffffe30:  00000860 3fff06e8 3fff0768 3ffffee0  
3ffffe40:  00000001 00000000 00000020 40100c6f  
3ffffe50:  00000000 00000000 4bc6a7f0 00000000  
3ffffe60:  00000001 00000000 3fff07a4 40215755  
3ffffe70:  00000000 00000010 00000020 40208f4f  
3ffffe80:  00000001 00000000 3fff07a4 402096b9  
3ffffe90:  00001780 000002f0 000002f0 401009e7  
3ffffea0:  00000860 3fff06e8 3fff0768 3fff09b4  
3ffffeb0:  00000001 00000000 00000020 40100c6f  
3ffffec0:  00000000 00000000 4bc6a7f0 00000000  
3ffffed0:  00000001 00000000 3fff07a4 40215755  
3ffffee0:  00000000 696d7568 79746964 40208f4f  
3ffffef0:  000012c8 00000259 00000259 401009e7  
3fffff00:  00000005 00000000 3fff07a4 3ffeec68  
3fffff10:  401054e5 00bd830c 3ffeecc8 00000000  
3fffff20:  3ffee230 3ffeecc8 00001388 3ffeecc8  
3fffff30:  3fffdad0 3ffeec68 4020c8a4 3fffefa0  
3fffff40:  3ffeecc8 3fffdad0 3ffeec68 4020d126  
3fffff50:  3fffdad0 41e80000 41c1999a 402049ac  
3fffff60:  302e3900 a9e00030 80fee800 3fff0600  
3fffff70:  001a001f 80fefed4 696d7500 79746964  
3fffff80:  80000000 00000000 00000001 40100230  
3fffff90:  3fffdad0 00000000 3ffeec28 40204701  
3fffffa0:  feefeffe 00000000 3ffeec28 4020c9c8  
<<<stack<<<

--------------- CUT HERE FOR EXCEPTION DECODER ---------------
```

While its not great that this error is being thrown, its not inhibiting any device usage and the configuration seems to be stored and loaded correctly.

**Comment by bocklucas** [comment]
Tentatively closing this issue, I believe this was caused by attempting to disable the soft Access Point (when in reality it wasn't running to begin with) in the `resetConfig` ISR.

Removing the line has seemed to stop the error.

**Comment by bocklucas** [comment]
Closed in #9
